packages/muelu/src/Utils/MueLu_AggregationExportFactory_def.hpp

```diff
@@ -418,7 +411,7 @@ namespace MueLu {
       {
         if(vertex2AggId_[j] == i)
         {
-          Point p(xCoords_[j], yCoords_[j]);
+          Point p(xCoords[j], yCoords[j]);
```

Comment:
Why were these changes needed?

Reply:
Because `AggregationExportFactory` stored long-term pointers to `coordinates->getData()` as members of the class, which caused issues when it comes time to transfer coordinates to the next level in MueLu. I changed it to instead store RCPs for the MultiVectors and only grab the data pointers when needed.